tutorials/how_to/elmo.md

```diff
@@ -53,10 +52,31 @@ JSON-serialized string with a mapping from sentences to line indices to the
 
 For more details, see `allennlp elmo -h`. 
```

Comment:
Let's also add somewhere in the "Writing contextual representations to disk" section that this is a thin wrapper around the `allennlp.commands.elmo.ElmoEmbedder` class that adds a command line interface.

Reply:
@matt-peters for a future PR...